toolkit/tools/pkg/imagecustomizerlib/artifactsinputoutput.go

```diff
@@ -24,47 +26,76 @@ const (
 	SystemdBootDir = "EFI/systemd"
 )
 
+type OutputArtifactMetadata struct {
+	Partition   InjectFilePartition `yaml:"partition"`
+	Destination string              `yaml:"destination"`
+	Source      string              `yaml:"source"`
+}
+
+type InjectFilePartition struct {
```

Comment:
Can probably reuse (and extend) `IdentifiedPartition` instead of adding a new type.

Reply:
@cwize1 , `IdentifiedPartition` has the `IdType` as `IdentifiedPartitionType` rather than `MountIdentifierType`. Which makes it easier for `findPartition()` to be used. Your thoughts?